os/net/netdev/netdev_ioctl.c

```diff
@@ -1149,6 +1149,47 @@ static int free_addrinfo(struct addrinfo *ai)
 	return 0;
 }
 
+#if defined(CONFIG_NET_LWIP_DHCP)
+#define DHCPD_MQ_NAME "dhcpd_queue"
+#define DHCPD_MQ_LEN 11
+#define DHCPD_MQ_MAX_LEN 20
+
+static void _dhcpd_join(void)
+{
+	printf("dhcpd joined");
+
+	struct mq_attr attr;
+	attr.mq_maxmsg = DHCPD_MQ_MAX_LEN;
+	attr.mq_msgsize = DHCPD_MQ_LEN;
+	attr.mq_flags = 0;
+	attr.mq_curmsgs = 0;
+
+	mqd_t md = mq_open(DHCPD_MQ_NAME, O_RDWR | O_CREAT, 0666, &attr);
+	if (!md) {
+		printf("mq open fail%s %d\n", __FUNCTION__, errno);
+		mq_close(md);
+		return;
+	}
+
+	char *msg = "dhcpd_join";
+	int mq_ret = mq_send(md, msg, DHCPD_MQ_LEN, 100);
+	printf("[shbyeon] send mq %s\n", msg);
```

Comment:
I already know who you are. also it is kernel, do not use printf.

Reply:
Applied. Thanks ^^;;